dd-sdk-android-ndk/src/main/cpp/model/crash-log.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+
+#include "crash-log.h"
+#include <string>
+#include "../utils/format-utils.cpp"
```

Comment:
Usually you do not put the `..` in the include path, instead you make sure that the proper include directory is listed in your makefile

Reply:
yes...that's a special case here as I am using the `cpp` file and this does not work directly for some reason :(. I cannot use the header because it seems it doesn't like the `template<typename... Args>` and throws an exception. Or maybe I do not know how exactly to add this template function definition in the header.